docs/source/glossary.rst

```diff
@@ -0,0 +1,83 @@
+==============
+Conda Glossary
+==============
+
+.. _condarc:
```

Comment:
There is a build error because there needs to be a blank line after each of these. Also Sphinx will fail unless this page is added to a toctree ("toctree" stands for "table of contents tree", i.e., it needs to be added to `index.rst` under one of the `toctree` directives).